services/core-web/src/components/modalContent/AddReclamationInvoiceModal.js

```diff
@@ -11,23 +16,43 @@ const propTypes = {
   mineGuid: PropTypes.string.isRequired,
   invoice: CustomPropTypes.invoice,
   edit: PropTypes.bool,
+  balance: PropTypes.string,
+  formValues: CustomPropTypes.invoice.isRequired,
 };
 
 const defaultProps = {
   invoice: {},
   edit: false,
+  balance: "",
 };
 
 export const AddReclamationInvoiceModal = (props) => {
-  const handleAddReclamationInvoice = (values) => {
-    const newValues = props.edit ? { values, ...props.invoice } : values;
+  const handleAddReclamationInvoice = (values) =>
     props.edit
       ? props.onSubmit(values, props.invoice.reclamation_invoice_guid)
       : props.onSubmit(values, props.permitGuid);
-  };
-
+  const isNegative = (value) => Math.sign(value) === -1;
+  const newBalance = props.formValues.amount ? props.balance - props.formValues.amount : 0;
+  const isNewBalanceNegative = isNegative(newBalance);
+  const showErrors = isNegative(props.balance) || isNewBalanceNegative;
```

Comment:
I think `const showErrors = props.balance < 0 || newBalance < 0` might be a bit more readable.